ph5/utilities/ph5validate.py

```diff
@@ -847,14 +831,32 @@ def get_args():
 
     args = parser.parse_args()
 
-    # Add formatter
-    args.level = str(args.level).upper()
-    if args.level != "ERROR" and args.level != "WARNING" and \
-            args.level != "INFO":
-        raise ValueError("Invalid logging level.")
+    outfile = 'ph5_validate.log' if args.outfile is None else args.outfile
+    PH5 = args.nickname
+    if not os.path.exists(PH5) and not os.path.exists(PH5 + '.ph5'):
+        LOGGER.error("{0} not found.".format(PH5))
+        sys.exit()
+    else:
+        # Set up logging
+        # Write log to file
+        ch = logging.FileHandler(os.path.join('.', outfile))
```

Comment:
This won't let a user set outfile to an absolute path, which should be allowed.

Reply:
Should I change this to:
`ch = logging.FileHandler(outfile)`

Reply:
yes, if no path I think it assumes ./